.github/CODEOWNERS

```diff
@@ -22,3 +22,4 @@
 /traffic_director/      @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /vpc/                   @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
 /managedkafka/          @terraform-google-modules/managedkafka-dev-team @terraform-google-modules/terraform-samples-reviewers
+/backupdr/              @terraform-google-modules/gcbdr-samples-team @terraform-google-modules/terraform-samples-reviewers
```

Comment:
FYI: Per GH this team (`@terraform-google-modules/gcbdr-samples-team`) doesn't (currently) exist.

Reply:
Not sure why github is throwing error, but team is present
https://opensource.corp.google.com/github/team?org=terraform-google-modules&team=2184949506716467201

Reply:
It was an access issue, I have granted access to that team for this repo now